CDP4ParameterSheetGenerator/Assemblers/OptionSheet/OptionSheetRowAssembler.cs

```diff
@@ -120,6 +120,29 @@ public void Assemble()
             foreach (var nestedElementExcelRow in nestedElementExcelRows)
             {
                 var deeprows = nestedElementExcelRow.GetContainedRowsDeep();
+
+                foreach (var itm in deeprows)
+                {
+                    var splitlevel = itm.ModelCode.Split('\\')[0].Split('.').Length;
```

Comment:
Is there really no other way to determine the level?

Reply:
There is a level property but always comes as zero. Other than that we do not have any thing that can be used directly.